samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/MainActivity.kt

```diff
@@ -45,6 +45,14 @@ open class MainActivity : AppCompatActivity(), LoginFragment.OnLoginCompleteList
                 Config.custom(CONFIG_URL).then { value: Config ->
                     val acct = FirefoxAccount(value, CLIENT_ID, REDIRECT_URL)
                     account = acct
+
+                    val pairingUrl = getIntent().getExtras().getString("pairingUrl") ?: ""
+                    if (pairingUrl != "") {
```

Comment:
pairingUrl can't be null here. So, this check shouldn't be necessary.

Reply:
from mtg: let's try to use `let` instead